beacon-chain/cache/depositcache/deposits_cache.go

```diff
@@ -54,6 +53,7 @@ type DepositCache struct {
 	pendingDeposits   []*dbpb.DepositContainer
 	deposits          []*dbpb.DepositContainer
 	finalizedDeposits *FinalizedDeposits
+	depositMap        map[[48]byte][]*dbpb.DepositContainer
```

Comment:
We tend to avoid giving variables names related to their type in Go. This doesn't tell us much about what the keys in the map are, so maybe we can call it depositsByPubKey